robotoff/app/core.py

```diff
@@ -363,6 +369,8 @@ def get_logo_annotation(
     if offset is not None:
         query = query.offset(offset)
 
+    # query = query.order_by(LogoAnnotation.image_prediction.timestamp.desc())
```

Comment:
here you can use LogoAnnotation.id for order_by. `id` exists because it's automatically added by peewee.

See https://docs.peewee-orm.com/en/latest/peewee/models.html#fields
> In the above example, because none of the fields are initialized with primary_key=True, an auto-incrementing primary key will automatically be created and named “id”. Peewee uses [AutoField](https://docs.peewee-orm.com/en/latest/peewee/api.html#AutoField) to signify an auto-incrementing integer primary key, which implies primary_key=True.